namespace cpp entity
namespace py ccentity.transport
namespace as3 com.dreamfire.sanguo.data.definition.entity

struct TransportSnapshotField {
  1 : required i64 actor_id_;               // 玩家ID
  2 : required string actor_name_;          // 玩家名称
}

